app/backend/src/couchers/metrics.py

```diff
@@ -39,35 +39,143 @@ def observe_in_servicer_duration_histogram(method, user_id, status_code, excepti
+    ]
+]
+
+users_gauge = _make_gauge_from_query(
+    "couchers_users", "Total number of users", select(func.count()).select_from(User).where(User.is_visible)
 )
-active_users_5m_gauge.set_function(_get_active_users_5m)
 
+man_gauge = _make_gauge_from_query(
+    "couchers_users_man",
+    "Total number of users with gender 'Man'",
+    select(func.count()).select_from(User).where(User.is_visible).where(User.gender == "Man"),
+)
 
-def _get_users():
-    with session_scope() as session:
-        return session.execute(select(func.count()).select_from(User).where(User.is_visible)).scalar_one()
+woman_gauge = _make_gauge_from_query(
+    "couchers_users_woman",
+    "Total number of users with gender 'Woman'",
+    select(func.count()).select_from(User).where(User.is_visible).where(User.gender == "Woman"),
+)
 
+nonbinary_gauge = _make_gauge_from_query(
+    "couchers_users_nonbinary",
+    "Total number of users with gender 'Non-binary'",
+    select(func.count()).select_from(User).where(User.is_visible).where(User.gender == "Non-binary"),
+)
 
-users_gauge = Gauge(
-    "couchers_users",
-    "Total number of users",
-    registry=main_process_registry,
+can_host_gauge = _make_gauge_from_query(
+    "couchers_users_can_host",
+    "Total number of users with hosting status 'can_host'",
+    select(func.count()).select_from(User).where(User.is_visible).where(User.hosting_status == HostingStatus.can_host),
+)
+
+cant_host_gauge = _make_gauge_from_query(
+    "couchers_users_cant_host",
+    "Total number of users with hosting status 'cant_host'",
+    select(func.count()).select_from(User).where(User.is_visible).where(User.hosting_status == HostingStatus.cant_host),
+)
+
+maybe_gauge = _make_gauge_from_query(
+    "couchers_users_maybe",
+    "Total number of users with hosting status 'maybe'",
+    select(func.count()).select_from(User).where(User.is_visible).where(User.hosting_status == HostingStatus.maybe),
+)
+
+completed_profile_gauge = _make_gauge_from_query(
+    "couchers_users_completed_profile",
+    "Total number of users with a completed profile",
+    select(func.count()).select_from(User).where(User.is_visible).where(User.has_completed_profile),
+)
+
+sent_message_gauge = _make_gauge_from_query(
+    "couchers_users_sent_message",
+    "Total number of users who have sent a message",
+    (
+        select(func.count()).select_from(
+            select(User.id)
+            .where(User.is_visible)
+            .join(Message, Message.author_id == User.id)
+            .group_by(User.id)
+            .subquery()
+        )
+    ),
+)
+
+sent_request_gauge = _make_gauge_from_query(
+    "couchers_users_sent_request",
+    "Total number of users who have sent a host request",
+    (
+        select(func.count()).select_from(
+            select(User.id)
+            .where(User.is_visible)
+            .join(HostRequest, HostRequest.surfer_user_id == User.id)
+            .group_by(User.id)
+            .subquery()
+        )
+    ),
+)
```

Comment:
I guess for both it would be interesting to have the opposite side to see if we have issues with superhosts (which I reckon we do)